.shippable.yml

```diff
@@ -55,6 +55,7 @@ build:
     - _make PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y CFG_TA_GPROF_SUPPORT=y CFG_FTRACE_SUPPORT=y CFG_SYSCALL_FTRACE=y CFG_ULIBS_MCOUNT=y
     - _make PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y CFG_VIRTUALIZATION=y
     - _make PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y CFG_CORE_SEL1_SPMC=y
+    - dd if=/dev/zero of=BL32_AP_MM.fd bs=2621440 count=1 && _make PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y CFG_STMM_PATH=BL32_AP_MM.fd CFG_RPMB_FS=y CFG_RPMB_FS_DEV_ID=1 CFG_CORE_HEAP_SIZE=524288
```

Comment:
Since we're compressing this in `scripts/gen_stmm_hex.py` we may want to add random instead.

Reply:
+1, this should use `/dev/urandom`

Reply:
Ok, I'll change that to /dev/urandom for now.
@jforissier since this commit is supposed to be the beginning of supporting "Secure partition", are we interested in more than compile testing this?
We can for example, run the whole thing under QEMU and make sure U-boot at can at least read and write variables.
Although far from complete, it's a better indication that the secure partition can at least launch StMM (for now). 
Once I merge the EDK2 missing parts, I can add the documentation on how to build OP-TEE with StMM support and the tests in shippable

Reply:
@apalos that would be great, but bear in mind that Shippable only allows 1h for Open Source project builds and we're already dangerously close to the limit.

Reply:
I can always provide prebuilt binaries for U-Boot and EDK2. The ABI is still not 100% set in stone, but we don't expect anything to change dramatically. In any case, if that changes I can send updates